remoteappmanager/docker/container.py

```diff
@@ -78,10 +78,10 @@ def from_docker_containers_dict(cls, docker_dict):
         """
         if docker_dict.get('Ports'):
             ip = docker_dict['Ports'][0].get('IP', "")
-            port = docker_dict['Ports'][0].get('PublicPort')
+            port = docker_dict['Ports'][0].get('PublicPort', 80)
         else:
             ip = ""
```

Comment:
Which brings me to the question... should we even fill this information at all? The defaults are already there...